tsconfig.build.json

```diff
@@ -5,15 +5,16 @@
     "noImplicitAny": true,
     "strictNullChecks": true,
     "noFallthroughCasesInSwitch": true,
-    "module": "commonjs",
-    "target": "es5",
-    "lib": ["es2015"],
+    "lib": ["ES2022"],
+    "module": "node16",
+    "target": "ES2022",
     "skipLibCheck": true,
     "types": [],
     "experimentalDecorators": true,
     "emitDecoratorMetadata": true,
     "sourceMap": true,
     "inlineSources": true,
-    "resolveJsonModule": true
+    "resolveJsonModule": true,
```

Comment:
l: since we no longer `require` the `package.json` to read the CLI version, can we get rid of `resolveJsonModule` (IIRC it defaults to `false`). 

edit: Actually not sure if we require a json somewhere else, so feel free to ignore

Reply:
Will do this in a follow up, already planned :)